python/packages/nisar/workflows/get_product_geometry.py

```diff
@@ -52,6 +52,20 @@ def get_parser():
                         choices=['A', 'B'],
                         help='Frequency band: "A" or "B"')
 
+    parser.add_argument('--spacing-y',
+                        dest='spacing_y',
+                        required=False,
+                        default=None,
+                        type=float,
+                        help='Y spacing, the same unit with the L2 product (default: None)')
```

Comment:
What if the input is an L1 product?

Not sure how the user of this interface is supposed to know what `default: None` means.

```suggestion
                        help='Y spacing, the same unit with the L2 product (only'
                        ' applicable if the input is a NISAR L2 product). Defaults'
                        ' to the Y-coordinate spacing of the product.')
```

Reply:
those parameters are implemented under the L2 product, so it will not have impact on the L1 product. In addition, L1 products grid normally do not use the x and y spacing convention, right? I like your change of the help information.